Carosa.Entities/VehicleUnit.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Carosa.Entities
+{
```

Comment:
The relation to Vehicle is missing (defined in the task)
FirstUsageAt should be a DatetimeOffset - it makes no sense as an int right?
Change NextService to NextServiceAt and to property type - what sense does it make as an int?
ReportedBroken as an int makes no sense as well. Maybe better name would be IsBroken.